notebooks/Viewing 3D volumetric data with matplotlib.ipynb

```diff
@@ -54,7 +61,10 @@
    },
    "outputs": [],
    "source": [
+    "# import data module \n",
```

Comment:
I would remove all comments about imports, as they almost repeat the line underneath them verbatim.